app/queries/my_facilities/missed_visits_query.rb

```diff
@@ -57,8 +57,9 @@ def missed_visit_totals
 
   def total_patients_per_facility
     @total_patients_per_facility ||=
-      facilities
-        .assigned_patients
```

Comment:
This scope results in a `WHERE patients IN (..list of all patients in those facilities)` query which is extremely expensive to generate. Going back to `.where(assigned_facility: facilities)` for performance.